pkg/cmd/create/create.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 var (
-	createLong    = "Create a new Brev machine"
```

Comment:
Appreciate the standardization of language. 

Minor follow up:
- Customer & internal documentation for product terminology